.codespellrc

```diff
@@ -0,0 +1,10 @@
+[codespell]
+skip = .git,*.pdf,*.svg,*.test,typeshed
+# fo,ot,statics,ans,alo,haa -- variables used
+# hax -- HAX, initials,
+# SEH,NAMS,WONT -- abbreviations but codespell is case incensitive
```

Comment:
I appreciate the irony of a typo in a .codespellrc file

```suggestion
# SEH,NAMS,WONT -- abbreviations but codespell is case insensitive
```

Reply:
yeah, Shame! ;-) Thanks for catching